src/components/codeContext.tsx

```diff
@@ -176,6 +176,65 @@ export async function fetchCodeKeywords(): Promise<CodeKeywords> {
   };
 }
 
+function getCsrfToken(cookies = document.cookie): string | null {
+  // is sentry-sc in production, but may also be sc in other envs
+  // So we just try both variants
+  const cookieNames = ['sentry-sc', 'sc'];
+
+  return cookieNames
+    .map(cookieName => getCookie(cookies, cookieName))
+    .find(token => token !== null);
+}
+
+function getCookie(cookies: string, cookieName: string) {
+  const csrfCookie = cookies.split(';').find(cookie => {
+    const str = cookie.trim();
+    return str.startsWith(`${cookieName}=`);
+  });
+
+  return csrfCookie ? csrfCookie.split('=')[1] : null;
+}
+
+export async function createOrgAuthToken({
```

Comment:
Is there a better place we can put this export? Doesn't feel right to have this inside codeContext.

I know this code is a bit of a mess right now, so it's probably unclear where this should go

Reply:
I can move it into the org auth token note component!

Reply:
There isn't really a utils kind of place as far as I've seen so far? Just the `components`, and this doesn't really have a dedicated component backing it 😬 it's only used in the `codeBlock` as of now. Happy to move it to any place you think makes sense!

Reply:
Leave it here for now, we can move it around later once this repo is a little better organized.